framework/src/base/Moose.C

```diff
@@ -218,6 +222,9 @@ addActionTypes(Syntax & syntax)
                            "(setup_mesh_complete)"
                            "(determine_system_type)"
                            "(create_problem)"
+                           "(create_problem_custom)"
+                           "(create_problem_default)"
+                           "(create_problem_complete)"
```

Comment:
So if I create a `[Problem]` block, it seems like the first action would always create the problem with this design. The custom task would run after that and wouldn't take effect. How does this work?

If the design is to allow parameters  from the `[Problem]` to be applied to a custom problem, I don't see how this is working. Should custom be first?